StoreManagementService/tests/PBJ.StoreManagementService.Business.UnitTests/CommentServiceTests.cs

```diff
@@ -0,0 +1,192 @@
+﻿using AutoFixture;
+using AutoMapper;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using PBJ.StoreManagementService.Business.Exceptions;
+using PBJ.StoreManagementService.Business.Mappers;
+using PBJ.StoreManagementService.Business.Services;
+using PBJ.StoreManagementService.DataAccess.Entities;
+using PBJ.StoreManagementService.DataAccess.Repositories.Abstract;
+using PBJ.StoreManagementService.Models.Comment;
+using PBJ.StoreManagementService.Models.User;
+using Xunit;
+
+namespace PBJ.StoreManagementService.Business.UnitTests
+{
+    public class CommentServiceTests
+    {
+        private readonly Mock<ICommentRepository> _mockCommentRepository;
+        private readonly IMapper _mapper;
+        private readonly Fixture _fixture;
+
+        public CommentServiceTests()
+        {
+            _mockCommentRepository = new Mock<ICommentRepository>();
+
+            _mapper = new Mapper(new MapperConfiguration(configuration =>
+            {
+                configuration.AddProfile<BusinessProfile>();
+            }));
+
+            _fixture = new Fixture();
+
+            _fixture.Behaviors.OfType<ThrowingRecursionBehavior>().ToList()
+                .ForEach(b => _fixture.Behaviors.Remove(b));
+            _fixture.Behaviors.Add(new OmitOnRecursionBehavior());
+        }
+
+        [Theory]
+        [InlineData(2)]
+        public async Task GetAmountAsync_ShouldReturnListOfCommentDto(int amount)
```

Comment:
Read about GWT method namings for tests

Reply:
Yes, Given-When-Then (Add_WhenGivenTwoPositiveIntegers_ReturnsSum). Please fix the naming and other things looks good.